docs/UserGuide.md

```diff
@@ -95,17 +100,42 @@ _Details coming soon..._
 
 **Command succeeds:** _Logout successful._
 
-### Change Password `[Coming Soon in v1.3]`
+### Account Recovery
 
-_Details coming soon..._
+**Format:** `recover account [--answer ANSWER --password NEW_PASSWORD --confirmPass CONFIRM_PASSWORD]`
 
-### Update details `[Coming Soon in v1.3]`
+**Example 1:** `recover account`
+**Example:** `recover account [--answer Koko --password newPassword123 --confirmPass newPassword123]`
```

Comment:
Would it be `Example 2` and on a new line? Also, would it be better not to show `[` and `]` as it is an example.

Reply:
Good spot, thanks!